### DISCUSSION TITLE
Remove a native div and add a component into the login.stories.tsx

### DISCUSSION THREAD

**Discussion by Ayuro** [keycloakify/keycloakify#712] [Q&A]
Hello there,

I'm customized the pages I needed and checked them with storybooks.
I add an `<header>` element with div and logo etc (stored in a headerRCC.tsx that call a rccLogo.tsx, and I import it into the login.stories.tsx
I also set the display on the div containing the title "My Realm" to none as I don't need it ( id: kc-header ).

So when I start storybooks, everything is fine, I do not have the "My Realm" title and my header is there displaying the logo.

But when I get to the next step, using the "npx keycloakify start-keycloak" command, the "My Realm" div is still on display none, so ok.
But the element header I add and whose are displayed with storybooks or not with Vite ...

So, how could I remove the "My Realm" div definitely (if it's even possible), and what I do wrong in the adding of my element?

Can you help please? Sorry if it's not clear, I stay at your disposal for any questions.

This is the repo: https://github.com/Ayuro/kcify-starter.git

PS: At first rccLogo.tsx was stored in a folder called atoms and headerRCC.tsx was in a folder called molecules. has Vite didn't render it properly I thought it was because of multi folders thing. So I moved them all into the same folder as Login.stories.tsx, and adapt the imports, but that didn't changed anything.

**Comment by garronej** [comment]
Hello @Ayuro,  

There is a missunderstanding of what Storybook is about.  
Storybook is a tool to enable you to test your pages outside of keycloak this is not where you develop your pages.  
The only thing to customize in the stories file are the mock kcContext provided.  

Whay you want to do is eject the Template.tsx and modify it: `npx keycloakify eject-page` and select `Template.tsx`.  

https://docs.keycloakify.dev/customization-strategies/component-level-customization/in-react-components  

Another point is I saw you targeted classes like `.login-pf` thoses are Paternfly classes (the default styles), if you decide later you want to disable patternfly it will break all your styles. It can also break when you update.  
You should only target classes that start with `kc` like `kcHtmlClass`.  

https://docs.keycloakify.dev/customization-strategies/css-level-customization

**Reply by Ayuro** [reply]
Oh ... Thanks you, I'll look at it right away and come back to you if I've another question or to close this topic.
